CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## New Features
 
 ## Improvements
+- PR #2637: Using RAFT comms in cuML
```

Comment:
now that we have decided to do a monolithic PR, please update this message for posterity purposes.